diesel_cli/src/print_schema.rs

```diff
@@ -246,17 +246,30 @@ impl Display for CustomTypeList {
         if self.types.is_empty() {
             return Ok(());
         }
+        let mut out = None;
```

Comment:
How about we just create the module here since we just checked that `!self.types.is_empty()`?
That would remove the need for `out` being an Option and for the `first` stuff.
That changes the behaviour for non-postgres backends but they `unreachable!()` anyway, and I'm not even sure we can really say it's most likely they wouldn't need an `sql_types` module should we ever give them some form of support for custom types.